app/assets/stylesheets/container_dashboard.scss

```diff
@@ -0,0 +1,913 @@
+//
+// Containers Dashboard
+// --------------------------------------------------
+.container-heatmap-card {
+  padding-top: 10px;
+  width: 100%;
+}
+
+.usage-heatmap-body {
+  padding: 0;
+}
+
+.card-pf .loading {
+  margin: 10px auto;
+}
+
+.card-pf-status .card-pf-title .pficon {
+  color: #1186C1 !important;
+}
+
+.card-pf-status {
+
+  .card-pf-title {
+    margin: 0;
+    padding: 10px 0;
+    text-align: center;
+
+    .card-pf-status-count {
+      display: block;
+      font-size: 24px;
+      vertical-align: bottom;
+      text-align: center;
+      line-height: 28px;
+    }
+  }
+
+  & a {
+    color: #000000;
+    text-decoration: none;
+  }
+
+  & a:hover {
+    color: #00618a;
+    text-decoration: underline;
+  }
+
+  .pficon {
+    position: absolute;
+    left: 10px;
+    top: 50%;
+    margin-top: -20px;
+    width: 54px;
+    font-size: 26px;
+  }
+
+  .indicator {
+    font-size: 20px;
+
+    .count {
+      top: -2px;
+      position: relative;
+      padding-left: 5px;
+    }
+  }
+
+  .status {
+    position: absolute;
+    right: 15px;
+    top: 50%;
+    margin-top: -15px;
+    width: 54px;
+    font-size: 20px;
+  }
+}
+
+.card-pf-providers {
+  padding-top: 10px;
+
+  .providers {
+    padding: 20px 0;
+
+    & a {
+      color: #000000;
+      text-decoration: none;
+    }
+
+    & a:hover {
+      color: #00618a;
+      text-decoration: underline;
+    }
+    .count {
+      margin: 0 5px;
+    }
+
+    .provider-icon {
+      font-size: 22px;
+      position: relative;
+      top: 5px;
+      padding-right: 5px;
+      opacity: .9;
+
+      &:hover {
+        opacity: 1;
+      }
+    }
+  }
+
+  .provider-icon-large {
+    font-size: 100px;
+    position: relative;
+    padding-bottom: 14px;
+  }
+}
+
+.card-heatmap-body {
+  padding: 0;
+}
+
+
+
+.aggregate-status-separator {
+  width: 15px;
+}
+
+.card-pf.card-pf-aggregate-status {
+  min-height: 144px;
+}
+
+.providers-status {
+  padding-bottom: 9px !important; }
+
+.pf-obj-status-large {
+  .card-pf-title {
+    padding-top: 24px;
+    padding-bottom: 24px;
+  }
+}
+
+.pficon-kubernetes::before {
+  color: #326de6;
+}
+
+.pficon-openshift::before {
+  color: #da2430;
+}
+
+/////////////////////////////////////////////
+
+.container {
```

Comment:
Not sure about the stuff above, but I'm *pretty* sure `.container`, `.row`, `.col-xs-1`, etc. are all in patternfly already... We probably don't want it repeated here..?